### PR TITLE
Drone CI set-up

### PR DESCRIPTION
**Pull request by JoeChapman**
- add drone yaml
- remove travis yaml
- update Dockerfile
- add Dockerfile-acceptance
- test.sh and make executable
- remove Jenkins ci_build.sh

### CODE REVIEW

**Comment by JoeChapman** [comment]
I have switched off the travis hooks since I pushed this branch

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    commands:
+      - npm --loglevel warn install
+      - npm run test:ci
+
+  build_app:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker build -t rotm .
```
Comment (easternbloc) [inline .drone.yml]: I'm thinking as we'll do this so often we should just give it a tag of `-t app`

Comment (easternbloc) [inline .drone.yml]: Please add:

```
when:
      event: pull_request
```
as per https://github.com/UKHomeOffice/brp_app/blob/master/.drone.yml

This means we don't waste time building docker images for pushes just for PR's

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  build_acceptance:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker build -f Dockerfile-acceptance -t acceptance .
```
Comment (easternbloc) [inline .drone.yml]: and here

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  test_acceptance:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker run -e NODE_ENV=ci -d --net host rotm
+      - docker run --rm --net host acceptance
```
Comment (easternbloc) [inline .drone.yml]: and here

File: test.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+${PHANTOM_JS}/bin/phantomjs --webdriver=4444&
```
Comment (easternbloc) [inline test.sh]: I have another idea for this that I'd like to add to this PR this is starting to feel messy that we add phantom like this

File: .drone.yml (hunk 1 above)
Comment (joefitter) [inline .drone.yml]: @easternbloc if we remove the push build, trigger a build on a PR, then make changes to the branch and push - will it rebuild, as technically not a new PR. In these situations we would want a push to trigger a build, else we would have to cancel and resubmit the PR or trigger manually

Comment (easternbloc) [inline .drone.yml]: It'll rebuild. It just sees the event as a pr even when you add extra commits so all is good with the world.

**Pull request merged**